playbooks/templates/.env.j2

```diff
@@ -176,19 +176,20 @@ SKYNET_ACCOUNTS_LOG_LEVEL={{ webportal_common_config.accounts_log_level }}
 # - 'false': All users incl. anonymous users can access the portal (access to portal is public)
 # REQUIRED: manually defined by user, or default is set in docker-compose file
 ACCOUNTS_LIMIT_ACCESS={{ webportal_common_config.accounts_limit_access | string | lower }}
+{% endif %}
 
-{% if webportal_common_config.accounts_limit_access | string | lower != 'false' %}
-# Account Test account for health checks.
+{% if webportal_common_config.accounts_limit_access | default('') | string | lower != 'false' %}
+# Account Test account for health checks and integration tests.
 #
 # REQUIRED when ACCOUNTS_LIMIT_ACCESS is not 'false': manually defined by user.
 # API key takes precedence over email and password. It is recommended.
-# Email and password are deprecated and will be removed in the future.
+# Email and password are deprecated in use for the health checks but are needed
```

Comment:
I think the comment here was valid. We should be able to remove ACCOUNTS_TEST_USER_EMAIL and ACCOUNTS_TEST_USER_PASSWORD since we don't need to store it once we have the api key in ACCOUNTS_TEST_USER_API_KEY.

Reply:
Some service is still failing for it being required. Need to find that service and fix that.

Reply:
We should update the health check repo because API keys are usable for its purposes. We should double-check but I believe the only thing it needs is to verify the tier of the user, which it can do by calling `GET /user/limits`, which supports API keys.

Reply:
Created a follow up https://linear.app/skynetlabs/issue/SKY-828/remove-test-email-and-test-password-from-env-file

Shouldn't block this PR.